lib/main_containers.pm

```diff
@@ -159,37 +159,38 @@ sub load_host_tests_podman {
 
 sub load_host_tests_docker {
     my ($run_args) = @_;
+    my $docker = get_var("DOCKER_RUNTIME", "docker");
```

Comment:
I would much rather create a `load_hosts_tests_nerdctl` than adding here switches. A little bit of redundancy is IMHO better than increased complexity, especially if the complexity is at cost of the more important test runs.